lib/uri_service/solr_params.rb

```diff
@@ -0,0 +1,42 @@
+module URIService
+  class SolrParams
+    DEFAULT_ROWS = 25
+
+    attr_reader :parameters
+
+    def initialize
+      @parameters = {
+        q: nil,
+        qt: 'search',
+        fq: [],
+        rows: DEFAULT_ROWS,
+        start: 0
+      }
+    end
+
+    def fq(field, value)
+      @parameters[:fq] << "#{field}:\"#{value}\"" unless value.nil? ## should probably escape
+      self
+    end
+
+    def rows(num)
+      @parameters[:rows] = num
+      self
+    end
+
+    def q(query)
+      @parameters[:q] = query unless query.nil?
```

Comment:
Maybe we want to escape q with `URIService.solr.solr_escape(q)`?  We should think/talk about that.

Reply:
yea, i think we talked about it earlier, we should be escaping unless we want to allow users more control over querying. I forgot to add that in